### PR TITLE
Exo22

### PR DESCRIPTION
**Pull request by tianyuchen**

### CODE REVIEW

File: 022-Names_scores.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# For example, when the list is sorted into alphabetical order, COLIN, which is
+# worth 3 + 15 + 12 + 9 + 14 = 53, is the 938th name in the list. So, COLIN
+# would obtain a score of 938 × 53 = 49714.
+#
+# What is the total of all the name scores in the file?
+
+def alphabeticalValue(name):
+    value = 0
+    for character in name.lower():
+        position = ord(character) - 96
```
Comment (gameboy1024) [inline 022-Names_scores.py]: Leave a comment:
// 96 == ord('a')

File: 022-Names_scores.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+def nameScore():
+    sum = 0
+    with open('022_names.txt') as f:
+        content = f.readlines()
+        for line in content:
+            names = line.replace('"', '').split(',')
+            names.sort()
+    for name in names:
+        sum += alphabeticalValue(name) * (names.index(name) + 1)
```
Comment (gameboy1024) [inline 022-Names_scores.py]: names.index is very costy and unnecessary. Search "enumerate"

Comment (tianyuchen) [reply 022-Names_scores.py]: changed

File: 022-Names_scores.py (hunk 1 above)
Comment (tianyuchen) [reply 022-Names_scores.py]: ok, c'est fait

**Pull request merged**